lib/apr/views/commerce/commerce_transaction_slack_view.ex

```diff
@@ -1,120 +1,147 @@
 defmodule Apr.Views.CommerceTransactionSlackView do
   import Apr.Views.Helper
   alias Apr.Views.CommerceHelper
-  alias Stripe.PaymentIntent
+
+  @payments Application.get_env(:apr, :payments)
 
   def render(event, _routing_key) do
     order = event["properties"]["order"]
     seller = CommerceHelper.fetch_participant_info(order["seller_id"], order["seller_type"])
     buyer = CommerceHelper.fetch_participant_info(order["buyer_id"], order["buyer_type"])
 
-    fields =
-      basic_fields(event, buyer)
-      |> append_seller_admin(seller)
-      |> append_stripe_fields(
-        event["properties"]["external_id"],
-        event["properties"]["external_type"]
-      )
-      |> append_fulfillment_info(order)
+    attachments =
```

Comment:
before we had one Slack _attachment_ which had many fields, here we now use 3 separate attachments which will have their own fields and attributes. We separated them by, participant/order/transaction.